### PR TITLE
styling completed! ready for code review and integration with team components

### PR DESCRIPTION
**Pull request by stevens-works**

### CODE REVIEW

**Review by shakabandai — APPROVED** [review]
lgtm

File: package.json [hunk 1]
```diff
@@ -22,15 +22,17 @@
...
+    "styled-components": "^4.1.3"
   },
   "devDependencies": {
     "babel-core": "^6.23.1",
     "babel-loader": "^6.3.2",
     "babel-preset-es2015": "^6.22.0",
     "babel-preset-react": "^6.23.0",
     "eslint-config-hackreactor": "git://github.com/reactorcore/eslint-config-hackreactor.git",
-    "webpack": "^2.2.1"
+    "webpack": "^2.2.1",
```
Comment (therobinkim) [inline package.json:34]: webpack is v4 right now

File: server/index.js [hunk 2]
```diff
@@ -20,10 +20,17 @@ app.get('/', (req, res) => {
...
+  min = Math.ceil(min);
+  max = Math.floor(max);
+  return Math.floor(Math.random() * (max - min + 1)) + min;
+};
+
 app.get('/api/songs-info', (req, res) => {
   db.SongsInfo.findAll({
     where: {
-      id: 1
+      id: ranNum(0, 10)
```
Comment (therobinkim) [inline server/index.js:33]: this seems super random.. shouldn't we handle GET requests for `/api/songs-info/:id`?

**Comment by therobinkim** [comment]
imo better title for this pull request would be `complete styling` or `styling completed`.

`ready for code review` is something you would tell your teammates via Slack and/or w/in your PR comment

**Pull request merged**